### PR TITLE
🎉 Source HubSpot: support list memberships in Contacts stream

### LINKED ISSUE #8477: Source HubSpot: does not support "List Memberships" in contacts stream
## Tell us about the problem you're trying to solve
The HubSpot contacts stream went with default &showListMemberships=false, so it cannot retrieve list memberships in the contact object. The list membership info is required to describe the relationship between contact and contact list (basically, which list/lists this contact is assigned to).
To know which list a contact resides in is rather important in some use cases, it will be a great improvement to have it in the connector. If I did not remember incorrectly, Fivetran has this turned on in their HubSpot connector.

## Describe the solution you’d like
Support extra attribute "list-memberships" in the contact schema.

## Describe the alternative you’ve considered or used
There seems to be no alternative.

## Additional context
Related document: [https://legacydocs.hubspot.com/docs/methods/contacts/get_contacts](https://legacydocs.hubspot.com/docs/methods/contacts/get_contacts)

> 

List Memberships | &showListMemberships=x | Boolean "true" or "false" to indicate whether current list memberships should be fetched for the contact. Default is false.
-- | -- | --

## Are you willing to submit a PR?
Not at the moment.

### PR DESCRIPTION
**Pull request by vitaliizazmic**
## What
Supporting of list memberships in Contacts stream was implemented.

Closes #8477 

## How
According to the issue this feature is supported in API v1 by setting parameter showListMemberships=true in get all contacts endpoint. API will return list memberships for each contact record. But for syncing Contacts stream in connector API v3 is used, where list memberships for contacts isn't supported. Therefore, new stream was created based on get all contacts endpoint of API V1.
   
- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### CODE REVIEW

**Review by eliziario — APPROVED** [review]
Hey Vitally, the code LGTM but please check airbyte-oauth/src/main/java/io/airbyte/oauth/flows/HubspotOAuthFlow.java
in the method getScopes() to be sure the oauth flow asks for the scope needed for this object too.

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/contacts_list_memberships.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "canonical-vid": {
+      "type": ["null", "integer"]
```
Comment (sergei-solonitcyn) [inline airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/contacts_list_memberships.json:6]: Could it really be `null` (or absent)?

**Comment by vitaliizazmic** [comment]
@eliziario crm.objects.contacts.read is presented in requested scopes. It is enough for receiving Contacts, from which memberships lists are taken.

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/contacts_list_memberships.json (hunk 1 above)
Comment (vitaliizazmic) [reply airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/contacts_list_memberships.json:6]: @sergei-solonitcyn there is not enough information about it in documentation. In another streams null value is supposed.

**Pull request merged**